driver/bpf/types.h

```diff
@@ -216,6 +216,18 @@ struct scap_bpf_per_cpu_state {
 	struct tail_context tail_ctx;
 	unsigned long long n_evts;
 	unsigned long long n_drops_buffer;
```

Comment:
I would add a comment to indicate that this is the total number of buffer drops and includes all the other drop categories, and it is the fallback class of drops when we don't know how to label them. For us it is obvious but maybe in the future new contributors will appreciate it, same for `kmod`

Reply:
IMHO the renaming is even better!